src/ipc-handlers.ts

```diff
@@ -106,13 +106,17 @@ export async function getSiteDetails( _event: IpcMainInvokeEvent ): Promise< Sit
 	return mergeSiteDetailsWithRunningDetails( sites );
 }
 
-export function getInstalledApps(): InstalledApps {
+export function getInstalledAppsAndTerminals(): InstalledApps & InstalledTerminals {
```

Comment:
It seems to me that the `InstalledApps` and `InstalledTerminals` types are always used together now. Maybe we should just merge them in `src/ipc-types.d.ts`?

Reply:
That's a good point, thanks. I updated it in [2735521](https://github.com/Automattic/studio/pull/1259/commits/2735521498b95d7ba3a57d2623f829c2215e42a2)